data/language/nl-NL.yml

```diff
@@ -1849,13 +1849,13 @@ strings:
   1811: Bouwt {STRINGID} in de buurt van {STRINGID}
   1812: Bouwt {STRINGID} in de buurt van {STRINGID}
   1813: Bouwt {STRINGID} in de buurt van {STRINGID}
-  1814: Checking services near {STRINGID}
-  1815: Surveying landscape near {STRINGID}
+  1814: Onderzoekt de voorzieningen in de buurt van {STRINGID}
```

Comment:
Past deze string in het venster? Ik denk dat het net te klein is ervoor. In #359 ben ik 't company-venster aan het implementeren, dus op termijn kunnen we 't vergroten, maar toch.

Reply:
Dat zou inderdaad een probleem kunnen worden. Is dit wat:

```suggestion
  1814: Onderzoekt voorzieningen rondom {STRINGID}
```
?

Reply:
Vermoedelijk past dat wel, ja… Heb je 't al zelf kunnen testen in-game?

Reply:
Nu wel: het past. Het was even puzzelen om de string te vinden. Hij verschijnt als je het venster openklikt van een voertuig.

In dat licht bezien is “voorzieningen” geen goede vertaling, denk ik. Dan is “diensten” of “dienstregeling” beter. Wat denk jij? Die laatste is naar mijn idee het duidelijkst.

![locoScreenshot1](https://user-images.githubusercontent.com/1478678/69502118-dfadfd00-0f0b-11ea-95af-7f8420340f93.png)